python/vyos/frr.py

```diff
@@ -210,9 +247,15 @@ def reload_configuration(config, daemon=None):
     LOG.debug(f'reload_configuration: Executing command against frr-reload: "{cmd}"')
     output, code = popen(cmd, stderr=STDOUT)
     f.close()
+    emsg = ''
     for i, e in enumerate(output.split('\n')):
+        if e in bgp_vty_err_msg:
+            emsg = e
         LOG.debug(f'frr-reload output: {i:3} {e}')
     if code == 1:
+        if emsg:
+            raise ConfigError(emsg)
+
         raise CommitError('FRR configuration failed while running commit. Please ' \
                           'enable debugging to examine logs.\n\n\n' \
                           'To enable debugging run: "touch /tmp/vyos.frr.debug" ' \
```

Comment:
I don't think we should make people create files by hand. It's trivial to add an operational mode command that does it, I think we should do it.